fdbserver/ClusterController.actor.cpp

```diff
@@ -254,6 +257,17 @@ ACTOR Future<Void> clusterWatchDatabase(ClusterControllerData* cluster,
 			    .detail("ChangeID", dbInfo.id);
 			db->serverInfo->set(dbInfo);
 
+			if (SERVER_KNOBS->ENABLE_ENCRYPTION && !recoveredDisk) {
+				// EKP singleton recruitment waits for 'Master/Sequencer' recruitment, execute wait for
+				// 'recoveredDiskFiles' optimization once EKP recruitment is unblocked to avoid circular dependencies
+				// with StorageServer initialization
```

Comment:
```suggestion
				// EKP singleton recruitment waits for 'Master/Sequencer' recruitment, execute wait for
				// 'recoveredDiskFiles' optimization once EKP recruitment is unblocked to avoid circular dependencies
				// with StorageServer initialization
				// The waiting for recoveredDiskFiles is to make sure the worker server on the same process  has been registered with the new CC before recruitment.
```

Reply:
It probably needs to be formatted..

Reply:
Interesting, needs to check my formatting setting. Thanks for pointing it out.